### ISSUE TITLE
the plugin example doesn't appear to be working 

### ISSUE THREAD

**Issue by acgreek** [yoheimuta/protolint#260]
I cloned the repo and built the plugin and runtime 

```
$ make build/example/plugin
go build -o plugin_example _example/plugin/main.go
 make build/cmd/protolint
$ go build \
                -ldflags "-X github.com/yoheimuta/protolint/internal/cmd.version=`git describe --tags --abbrev=0` -X github.com/yoheimuta/protolint/internal/cmd.revision=`git rev-parse --short HEAD`" \
                -o protolint \
                cmd/protolint/main.go
$  cat test.proto 
syntax = "proto3";

package test;

enum Foo_bfooBob {
  TEST = 0;
  binBaz = 1;
  bop_Boo = 2;
}
$  ./protolint -plugin "./plugin_example" ./test.proto 
$
```
In reviewing your example plugin, as I understand it: "enumNamesLowerSnakeCaseRule" should be complaining because  " !strs.IsLowerSnakeCase(e.EnumName)" is true, but I get no error

note: I am using a Mac which of course has 'sh'

Sorry If I am doing something wrong

**Comment by acgreek** [comment]
issue was that I had ~/.protolint.yaml. One would think that adding the plugin would auto enable it.

**Comment by yoheimuta** [comment]
Thank you for reporting it.
For now, I added a note about this issue by https://github.com/yoheimuta/protolint/commit/5e872f529c04c3617a2dcc2e30d7cbd404545567.

If you have a better solution, let me know.
